lib/slax/slack.ex

```diff
@@ -100,4 +102,31 @@ defmodule Slax.Slack do
         :ok
     end
   end
+
+  def post_message_to_thread(%{text: text, channel: channel, thread_ts: thread_ts}) do
+    request =
+      URI.encode_query(
+        token: api_token(),
+        text: text,
+        channel: channel,
+        thread_ts: thread_ts,
+        unfurl_links: false,
+        unfurl_media: false
+      )
+
+    {:ok, response} =
+      Http.post(
+        "#{api_url()}/chat.postMessage",
+        request,
+        "Content-Type": "application/x-www-form-urlencoded"
+      )
+
+    case response do
```

Comment:
can we put the `Http.post` into the case so if it doesn't return `{:ok..` we can handle it gracefully?